cmd/img/main.go

```diff
@@ -0,0 +1,35 @@
+// Package main loads a very basic Hello World graphical application
+package main
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/app"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/cmd/fyne_demo/data"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/widget"
+)
+
+func main() {
```

Comment:
I think we could keep this in if it was able to showcase the different scale modes, maybe add a button so the user can flip through the different modes?

Reply:
Or maybe this should be integrated into fyne_demo?

Reply:
Yeah if it needs to be kept then pop in graphics demo - we don’t need another App I don’t think

Reply:
I was committed by mistake ;-)